src/ServiceControl.Monitoring/Http/Diagrams/MonitoredEndpointDigest.cs

```diff
@@ -0,0 +1,9 @@
+namespace ServiceControl.Monitoring.Http.Diagrams
+{
+    using System.Collections.Generic;
+
+    public class MonitoredEndpointDigest
+    {
+        public Dictionary<string, MonitoredEndpointMetricDigest> Metrics { get; set; } = new Dictionary<string, MonitoredEndpointMetricDigest>();
```

Comment:
This allows extensibility on the server side, which is great. I'm thinking now, should we write a test for public http API that would ensure that right metrics are in there? Just a thought.

Reply:
@Scooletz I like the idea. I think we could test other dictionaries (inside instances, metric-types and endpoint) as well. Separate PR?